pkg/resources/validation/namespace/namespace.go

```diff
@@ -38,17 +39,49 @@ func (v *Validator) GVR() schema.GroupVersionResource {
 }
 
 // Operations returns list of operations handled by this validator.
-func (v *Validator) Operations() []admissionregistrationv1.OperationType {
-	return []admissionregistrationv1.OperationType{
-		admissionregistrationv1.Update,
-		admissionregistrationv1.Create,
+func (v *Validator) Operations() []admv1.OperationType {
+	return []admv1.OperationType{
+		admv1.Update,
+		admv1.Create,
 	}
 }
 
 // ValidatingWebhook returns the ValidatingWebhook used for this CRD.
-func (v *Validator) ValidatingWebhook(clientConfig admissionregistrationv1.WebhookClientConfig) *admissionregistrationv1.ValidatingWebhook {
+func (v *Validator) ValidatingWebhook(clientConfig admv1.WebhookClientConfig) []admv1.ValidatingWebhook {
 	// Note that namespaces are actually CLUSTER scoped
-	return admission.NewDefaultValidatingWebhook(v, clientConfig, admissionregistrationv1.ClusterScope)
+
+	// standardWebhook validates all operations other than create on all namespaces.
+	standardWebhook := admission.NewDefaultValidatingWebhook(v, clientConfig, admv1.ClusterScope, []admv1.OperationType{admv1.Update})
+
+	// default configuration for all create operations except those belonging to the kube-system namespace
+	createWebhook := admission.NewDefaultValidatingWebhook(v, clientConfig, admv1.ClusterScope, []admv1.OperationType{admv1.Create})
+	createWebhook.Name = admission.CreateWebhookName(v, "create-non-kubesystem")
+	createWebhook.NamespaceSelector = &metav1.LabelSelector{
+		MatchExpressions: []metav1.LabelSelectorRequirement{
+			{
+				Key:      corev1.LabelMetadataName,
+				Operator: metav1.LabelSelectorOpNotIn,
+				Values:   []string{"kube-system"},
+			},
+		},
+	}
+
+	// kubeSystemOnlyWebhook is a separate webhook configuration that routes to this handler only if the namespace is equal to kube-system.
+	// This configuration differs from above because it allows create request to go through while the webhook is down if and only if the namespace is kube-system
```

Comment:
```suggestion
	// This configuration differs from above because it allows create request to go through while the webhook is down if and only if the namespace is kube-system.
```

Reply:
Fixed.